xml/deployment_installation.xml

```diff
@@ -825,14 +825,350 @@
+         </varlistentry>
+         <varlistentry>
+          <term><literal>admin</literal>/master/worker_flavor</term>
+          <listitem>
+           <para>
+            Select the machine flavor for your nodes
+           </para>
+          </listitem>
+         </varlistentry>
+         <varlistentry>
+          <term><literal>worker_count</literal>
+          </term>
+          <listitem>
+           <para>
+            Number of worker nodes to be launched
+           </para>
+          </listitem>
+         </varlistentry>
+         <varlistentry>
+          <term><literal>external_net</literal>
+          </term>
+          <listitem>
+           <para>
+            Select an external network that your cluster will be reachable from
+           </para>
+          </listitem>
+         </varlistentry>
+         <varlistentry>
+          <term><literal>internal_net_cidr</literal>
+          </term>
+          <listitem>
+           <para>
+            The internal network range to be used inside the cluster
+           </para>
+          </listitem>
+         </varlistentry>
+         <varlistentry>
+          <term><literal>dns_nameserver</literal>
+          </term>
+          <listitem>
+           <para>
+            Internal nameserver for the cluster
+           </para>
+          </listitem>
+         </varlistentry>
+        </variablelist>
+        <informalfigure>
+         <mediaobject>
+          <imageobject role="fo">
+           <imagedata fileref="horizon_stack_options.png" width="100%"/>
+          </imageobject>
+          <imageobject role="html">
+           <imagedata fileref="horizon_stack_options.png" width="100%"/>
+          </imageobject>
+         </mediaobject>
+        </informalfigure>
+       </step>
+       
+       <step>
+        <para>
+        Click <guimenu>Launch</guimenu>.
+        </para>
+       </step>
+       
+       <step>
+        <para>
+         After the cluster has been started and the cluster overview shows 
+         <literal>Create Complete</literal>, you need to find the external IP 
+         address for the admin node of your cluster (here: 
+         <literal>10.86.1.72</literal>). Now visit that IP address in your browser. 
+         You should see the &dashboard; login page and can continue with 
+         <xref linkend="sec.caasp.installquick.conf"/>.
+        </para>
+        
+        <informalfigure>
+         <mediaobject>
+          <imageobject role="fo">
+           <imagedata fileref="horizon_stack_resources.png" width="100%"/>
+          </imageobject>
+          <imageobject role="html">
+           <imagedata fileref="horizon_stack_resources.png" width="100%"/>
+          </imageobject>
+         </mediaobject>
+        </informalfigure>
+       </step>
+       
+      </procedure>
+    </sect3>
+      
+      <sect3>
+       <title>Using the &ostack; CLI</title>
+       <note>
+        <para>
+         You need to have access to the &ostack; command-line 
```

Comment:
80 chars?

Reply:
The link itself is 99 chars long, what can I do?